### ISSUE TITLE
Calibre Icons

### ISSUE THREAD

**Issue by leoheck** [leoheck/numix-calibre#3]
Progress of calibre icons to be made/matched with numix style.

| Icon | Numix Replacement |
| --- | --- |
| add_book |  |
| book |  |
| clear_left | edit-clear-locationbar-rtl |
| config | configure |
| connect_share |  |
| convert | gtk-convert |
| cover_flow |  |
| dialog_question | dialog-question |
| donate |  |
| edit-clear | edit-clear |
| edit_input |  |
| grid |  |
| help | help |
| identifiers |  |
| languages |  |
| lt |  |
| news |  |
| notify |  |
| publisher |  |
| rating |  |
| save |  |
| search | search |
| search_add_saved |  |
| search_copy_saved |  |
| series |  |
| store |  |
| tags |  |
| trash |  |
| tweak |  |
| user_profile |  |
| view |  |

**Comment by andia89** [comment]
There are a lot more, right?

**Comment by leoheck** [comment]
Yes, but this should be the first. We could add the others later. What do you think?
If you know some good replacements for them, I test them right now.

**Comment by andia89** [comment]
clear_left -> edit-clear-locationbar-rtl
config->configure
convert->gtk-convert
dialog-question->dialog-question
edit-clear->edit-clear
help->help
search->search

There are probably more

**Comment by leoheck** [comment]
I don't have dialog-question in my system. Can you send me?
